qiskit_ibm_runtime/sampler.py

```diff
@@ -25,7 +25,7 @@
 
 
 class Sampler(BaseSampler):
-    """Class for interacting with Qiskit Runtime Sampler primitive service.
+    """Class for interacting with Qiskit Runtime Sampler primitive service. Additionally, the successor of deprecated :class:`~qiskit_ibm_runtime.IBMSampler`.
```

Comment:
```suggestion
    """Class for interacting with Qiskit Runtime Sampler primitive service. Additionally, the successor of deprecated ``IBMSampler`` class.
```
Link won't work when IBMSampler class is removed.